src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -783,6 +783,61 @@ public void TestProcessRecycledPid()
             Assert.True(foundRecycled);
         }
 
+        [Theory]
```

Comment:
This test is Unix specific, but you changed the Windows implementation is well.  Is there no corresponding test on the Windows side?

Reply:
The Unix implementation has separate code paths for direct children vs other processes. For Windows, it is treated the same.

Reply:
In particular, how we detect process exits.